sdks/go/pkg/beam/forward.go

```diff
@@ -215,3 +216,5 @@ var (
 
 // EventTimeType is the reflect.Type of EventTime.
 var EventTimeType = typex.EventTimeType
+
```

Comment:
Exported types, even type aliases, require their own doc comment.

// MetricResult represents a single metric value, for use in writing predicate functions to query PipelineResults.